### PR TITLE
#issue30 SOS without opening app in emergency - added shortcut from homescreen

### PR DESCRIPTION
**Pull request by pawan2806**
## Related Issue 

-Added Quick Actions in app.

Closes: #30 

### Describe the changes you've made

SOS without opening app in emergency- added shortcut from homescreen.

## Type of change

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Mention any unusual behaviour of your code (Write NA if not)
NA

## Checklist:

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly whereever it was hard to understand.
- [x] My changes generate no new warnings.
- [x] Any dependent changes have been merged and published in downstream modules.

## Additional Info (optional)
Any additional information you want to give

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,7 +44,8 @@
 
 Considering the safety and security of women in India in the recent times, We wanted to give a try from our end to address the issue in a simpler and safer way.
 
-Looking at the recent trends and the most powerful weapon with the humanity - technology, we planned to use the same to give access to women in serious or dangerous situations to address the issue in a fast and easier way to ensure their security.
+Looking at the recent trends and the most powerful weapon with the humanity - technology, we planned to use the same to give access to women in serious or
```
Comment (hiashutoshsingh) [inline README.md:47]: Please revert this change. I think by mistake enter was pressed.

File: android/app/build.gradle [hunk 2]
```diff
@@ -25,7 +25,7 @@ apply plugin: 'com.android.application'
 apply from: "$flutterRoot/packages/flutter_tools/gradle/flutter.gradle"
 
 android {
-    compileSdkVersion 28
+    compileSdkVersion 29
```
Comment (hiashutoshsingh) [inline android/app/build.gradle:28]: Let's do this 30.

File: lib/services/googleAuth.dart [hunk 3]
```diff
@@ -30,6 +32,8 @@ class GoogleAuthenticate {
           var registeredUser =
               await FirebaseAuth.instance.signInWithCredential(credential);
 
+          // SnackBar snackBar= SnackBar(content: Text('Wait.... Process is going on'));
```
Comment (hiashutoshsingh) [inline lib/services/googleAuth.dart:35]: Remove this commented code. The below code for the snack bar looks good.

File: pubspec.lock [hunk 4]
```diff
@@ -0,0 +1,992 @@
+# Generated by pub
```
Comment (hiashutoshsingh) [inline pubspec.lock:1]: Please remove this lock file from commit, but keep this in your local.
See Why: https://dart.dev/guides/libraries/private-files#pubspeclock

File: Sheheroes_WatchOS/pubspec.lock [hunk 5]
```diff
@@ -63,7 +63,7 @@ packages:
       name: async
```
Comment (hiashutoshsingh) [inline Sheheroes_WatchOS/pubspec.lock:63]: Please remove this lock file from commit, but keep this in your local.
See Why: https://dart.dev/guides/libraries/private-files#pubspeclock

**Review by hiashutoshsingh — CHANGES_REQUESTED** [review]
Thanks, @pawan2806 for submitting PR.

There are some small changes required that would take only a little time of yours, then we are good to go.

File: lib/main.dart [hunk 6]
```diff
@@ -35,6 +44,41 @@ class _MyAppState extends State<MyApp> {
...
+
+    final QuickActions quickActions = QuickActions();
+    quickActions.initialize((String shortcutType) {
+
+      if (shortcutType == 'action_one') {
+        _service.sendSms(number);
+      } else {
+      _service.call(number);
+      }
+      setState(() {
```
Comment (hiashutoshsingh) [inline lib/main.dart:56]: Since we are not showing shortcut value anywhere in UI we can remove this code, but check after removing is it making a difference.

**Comment by pawan2806** [comment]
> Thanks, @pawan2806 for submitting PR.
> 
> There are some small changes required that would take only a little time of yours, then we are good to go.

@hiashutoshsingh I have made the suggested changes. Kindly review. Thanks.

**Comment by hiashutoshsingh** [comment]
> > Thanks, @pawan2806 for submitting PR.
> > There are some small changes required that would take only a little time of yours, then we are good to go.
> 
> @hiashutoshsingh I have made the suggested changes. Kindly review. Thanks.

LGTM!
Thanks for making a contribution. @pawan2806
